deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml

```diff
@@ -683,7 +683,7 @@ spec:
       #      - -c
       #      - >-
       #        mkdir -p /etc/elastic-agent/inputs.d &&
-      #        wget -O - https://github.com/elastic/elastic-agent/archive/8.10.tar.gz | tar xz -C /etc/elastic-agent/inputs.d --strip=5 "elastic-agent-8.10/deploy/kubernetes/elastic-agent-standalone/templates.d"
+      #        wget -O - https://github.com/elastic/elastic-agent/archive/8.10.tar.gz | tar xz -C /etc/elastic-agent/inputs.d --strip=5 "elastic-agent-main/deploy/kubernetes/elastic-agent-standalone/templates.d"
```

Comment:
Now trying to figure out where this change is coming from...

Reply:
kind of baffled by this, since I can't find anything that's doing some kind of find-replace for that...

Reply:
alright, so it's caused by this line in the release scripts: 
```
$(SED) -E -e "s#$(BASE)#$(RELEASE)#g" deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml; \
```

Not sure why CI wasn't choking on it before.

Reply:
yah, did a quick test, `make check-ci` should fail that.